### PR TITLE
ArchivesSpace: add "conditions governing access" note

### PR DESCRIPTION
**Pull request by mistydemeo**
This depends on artefactual/archivematica#408.

### CODE REVIEW

File: app/archivesspace/archivesspace.controller.js [hunk 1]
```diff
@@ -534,6 +555,7 @@ controller('ArchivesSpaceEditController', ['$uibModalInstance', 'levels', 'level
   vm.end_date = end_date;
   vm.date_expression = date_expression;
   vm.note = note;
+  vm.accessrestrict_note = note;
```
Comment (Hwesta) [inline app/archivesspace/archivesspace.controller.js]: Should this be accessrestrict_note?

File: app/archivesspace/archivesspace.controller.js [hunk 2]
```diff
@@ -116,8 +123,19 @@ controller('ArchivesSpaceController', ['$scope', '$uibModal', 'Alert', 'Archives
             return node.date_expression;
           },
           note: () => {
-            if (node.notes && node.notes[0]) {
-              return node.notes[0].content;
+            if (node.notes) {
+              let general_notes = node.notes.filter(note => note.type === 'odd');
+              if (general_notes.length > 0) {
+                return general_notes[0].content;
```
Comment (Hwesta) [inline app/archivesspace/archivesspace.controller.js:129]: Still only return the first one?

Comment (mistydemeo) [inline app/archivesspace/archivesspace.controller.js:129]: In the future we might change this, but right now the UI only supports a single note per type.

File: app/archivesspace/archivesspace.controller.js (hunk 1 above)
Comment (mistydemeo) [inline app/archivesspace/archivesspace.controller.js]: `accessrestrict` is the underlying name of this category of note in ArchivesSpace.

Comment (mistydemeo) [inline app/archivesspace/archivesspace.controller.js]: Oh wait, I misread!! Yeah, you're right.

File: app/archivesspace/archivesspace.controller.js (hunk 2 above)
Comment (Hwesta) [inline app/archivesspace/archivesspace.controller.js:129]: Ok, reasonable.

**Comment by Hwesta** [comment]
:+1:

**Pull request closed without merging**